### DISCUSSION TITLE
Can't get 'skip_condition' parameter to work with call service node

### DISCUSSION THREAD

**Discussion by mountaindude** [zachowj/node-red-contrib-home-assistant-websocket#676] [Q&A] [answered]
I'd like to start a HA service and have the service evaluate the conditions. 
I.e. setting skip_condition=false.

Can't get it to work though, the conditions are skipped no matter what I send from Node-Red to HA.
Tried setting a 'skip_condition' output as both type string and boolean, same result.

I'm prob just don't know how to use the node properly, but after failing to find any info on this in the docs I ask here.
Any ideas?

<img width="605" alt="image" src="https://user-images.githubusercontent.com/1029262/193419140-884645eb-b75d-41e1-91ce-7a3c51ee792b.png">

**Comment by zachowj** [comment]
I assume you want to trigger an events node in NR that has been exposed to HA. 

First, the domain should be `nodered`. Second, the `skip_condition` would go into the data field. `{"skip_condition": true}`

**Comment by mountaindude** [comment]
Uh, not sure I follow. 

A bit more context. I get electricity prices in Node-RED. 
Do some parsing on these and use a dedicated node for determining during which hours the heaters in the house should be turned on/off.
Turning the heaters on/off is done using a HA automation, which is started using "call service" nodes in Node-RED.

The flow below is a simplified version of the full setup, but captures the essence:

<img width="1097" alt="image" src="https://user-images.githubusercontent.com/1029262/193421913-8de18d31-b551-4417-9990-c83f470bb1a8.png">

The annotation in red ("Heating on") has a properties pane that looks as follows. 
"skip_condition" is mentioned there, I just don't understand where/how to use it.
So this is where I struggle. I'd like to start the HA automation, making HA taking the automation's conditions into account. 
These conditions look at the current temperature in a few different locations around the house. If it's already warm enough there's no reason to turn on the heaters.
If it's on the other hand too cold (below some threshold) in one or more locations, the automation should execute and ultimately starting the heaters.

This setup works, it's just that the automation always starts without first evaluating its conditions.

<img width="605" alt="image" src="https://user-images.githubusercontent.com/1029262/193421959-69680fc9-1667-4f70-b900-45b7ac8e7a84.png">

**Comment by zachowj** [comment, marked as answer]
I misunderstood what you were trying to do. You still want to put the `skip_condition` in the data field.

```
{"skip_condition": false}
```

![image](https://user-images.githubusercontent.com/37859597/193457100-07636e43-ea20-480b-ba2d-48755e7e9dd8.png)

**Comment by mountaindude** [comment]
💯 That did it. About the only variant I hadn't tried.. Thanks!
